test/e2e/helpers.go

```diff
@@ -693,15 +693,30 @@ func resolveKubetestRepoListPath(version string, path string) (string, error) {
 		return "", err
 	}
 
-	versionCutoff, err := semver.Make("1.21.0")
+	// - prior to K8s v1.21 repo-list-k8sprow.yaml should be used
+	//   since all test images need to come from k8sprow.azurecr.io
+	// - starting with K8s v1.25 repo lists repo list is not needed
+	// - use repo-list.yaml for everything in between which has only
+	//   some images in k8sprow.azurecr.io
+
+	v121, err := semver.Make("1.21.0")
 	if err != nil {
 		return "", err
 	}
 
-	if currentVersion.LT(versionCutoff) {
+	if currentVersion.LT(v121) {
```

Comment:
since 1.20 aged off we don't need this anymore?

Reply:
Good catch - will update

Reply:
I'll keep the comment about 1.21 for history - unless anyone wants that removed.

Reply:
updated